Dockerfile

```diff
@@ -3,6 +3,7 @@ MAINTAINER Marianne Tromp <mtromp@mtromp.com>
 
 USER root
 
+ENV DEBIAN_FRONTEND noninteractive
```

Comment:
Hey Marianne,

No, it definitely will not prevent the docker exec commands from working. I
put that in there just to suppress the excessive warnings about the
interactive front end. Feel free to eliminate that if it wasn't bothering
you!

Terry Yanko

(614) 592-6163

On Sun, May 22, 2016 at 7:27 PM, Marianne Tromp <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/mtromp/tdd-docker/pull/3#discussion_r64158308>:
>
> > @@ -3,6 +3,7 @@ MAINTAINER Marianne Tromp <mtromp@mtromp.com>
> >
> >  USER root
> >
> > +ENV DEBIAN_FRONTEND noninteractive
>
> What does this do? Will this prevent using the command: "docker exec -it
> lamp-stack bash"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mtromp/tdd-docker/pull/3/files/cdc94e2c6e0c6a2881a49a80892202e0bcb670f8#r64158308>
>